ગુજરાતી_પાઈથન/ભૂલ_અનુવાદક.py

```
@@ -59,14 +69,23 @@ def ગુજરાતી_એરર_મેળવો(self, exc_type, exc_value, e
+        # 3. Back-translate English keyword names embedded in the error message.
+        # e.g. NameError: name 'print' is not defined  →  "name 'છાપો' is not defined"
+        # Only keywords are back-translated; arbitrary identifiers stay as-is.
+        try:
+            from .અનુવાદક import _અનુવાદક
+            for eng_kw, guj_kw in _અનુવાદક.રિવર્સ_મેપ.items():
+                # Match 'keyword' with surrounding quotes inside the message
+                quoted = f"'{eng_kw}'"
+                if quoted in ગુજરાતી_મેસેજ:
+                    ગુજરાતી_મેસેજ = ગુજરાતી_મેસેજ.replace(quoted, f"'{guj_kw}'")
+        except Exception:
+            pass  # Never crash the error reporter itself
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)